app/src/docs/v1.api-spec.yaml

```
@@ -3455,6 +3457,119 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /forms/encryptionKey/algorithms:
+    get:
+      summary: Get the list of encryption algorithms
+      operationId: listEncryptionAlgorithms
+      security:
+        - BasicAuth: []
+        - BearerAuth: []
+          OpenID: []
+      tags:
+        - Form
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    code:
+                      type: string
+                      description: code for encryption Algorithm
+                    display:
+                      type: string
+                      description: display for encryption Algorithm
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /forms/{formId}/encryptionKey/{formEncryptionKeyId}:
+    get:
+      summary: Get the list of encryption algorithms
```

Should this be something like
```suggestion
      summary: Get the encryption key for a form
```